public/js/components/SourceTabs.js

```diff
@@ -12,6 +13,7 @@ const actions = require("../actions");
 const { isEnabled } = require("../../../config/feature");
 
 require("./SourceTabs.css");
+require("./Dropdown.css");
```

Comment:
If you want to extract it, a separate dropdown component would simplify this component also.